README.md

```diff
@@ -7,11 +7,53 @@ and copying out data at when a user requests it (the [`takeout` function]()).
 The functions are  flexible and easy to change to fit the needs of your specific
 users and apps.
 
-### How to play with these functions
+### How to contribute
 
 - [ ] Clone this repository
-- [ ] Create a new firebase project and go through the `firebase init` flow.
-This project uses Hosting, Functions, the RealTime Database, Firestore, and
-Storage.
+- [ ] Create a new firebase project and go through the `firebase init` flow. This project uses Hosting, Functions, the RealTime Database, Firestore, and Storage.
 - [ ] `cd` into the `functions` directory and `npm install`
 - [ ] Run `firebase deploy`
+
+
+### How to use these functions in your own project
+
+The `index.js` file has comments about how the functions work; this is about
+how to wire it up.
+
+The developer specifies the paths to data to wipeout or takeout. Those paths live in `user_privacy.json`. The data structures vary for each of the products:
+- For the RTDB, it’s a list of Strings to the path in the database of the form `"</users/uid/follows/..."`
```

Comment:
Suggest following [Google markdown style guide][1] which favors `*` over `-`, and calls for 4-space indent.  Also I suggest wrapping at 80 chars everywhere.

[1]: https://github.com/google/styleguide/blob/gh-pages/docguide/style.md#lists